packages/routes/README.md

```diff
@@ -0,0 +1,11 @@
+# routes
```

Comment:
Hmm, there's nothing left from old readme?

Reply:
I'll create a new one in the next PR. Obviously the new README will be somewhat based on what was in the older version, though currently `mkPath` is lacking many features from v1 so it will probably be much shorter

Reply:
👌